raycaster_float.cpp

```diff
@@ -19,6 +19,13 @@ bool RayCasterFloat::IsWall(float rayX, float rayY)
            (1 << (8 - (tileX & 0x7)));
 }
 
+float p2pDist(float x1, float y1, float x2, float y2)
```

Comment:
Can you pick up another function name which is self-explanatory? Use plain "distance" rather than "dist."

Reply:
Add `static` as the modifier of this function.